bench-vortex/src/bin/clickbench.rs

```diff
@@ -23,9 +24,18 @@ use vortex_datafusion::persistent::metrics::VortexMetricsFinder;
 
 feature_flagged_allocator!();
 
+#[derive(ValueEnum, Clone, Copy, Debug, Hash, Default, PartialEq, Eq)]
+enum Engine {
+    #[default]
+    DataFusion,
+    DuckDB,
+}
+
```

Comment:
in the command line, do I need to do `--engine duckdb` or `--engine DuckDB`? I think the former is better, and `Format` behaves that way (not sure if its the best way to achieve that, never got along with clap's docs).

Reply:
Right now as per Clap default it's:
```
      --engine <ENGINE>
          [default: data-fusion] [possible values: data-fusion, duck-db]
```
I'll change it to `datafusion` and `duckdb`. Good catch!